.github/workflows/update-third-party-notices.yml

```diff
@@ -0,0 +1,27 @@
+name: Check and Update Third Party Notices
+on: workflow_call
+
+jobs:
+  license-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install -g generate-license-file@1
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.BOT_REPO_SCOPED_TOKEN }}
+      - name: Generate third party notices for studio
+        run: cd packages/studio; ../../generate-third-party-notices.sh
+      - name: Generate third party notices for studio-plugin
+        run: cd packages/studio-plugin; ../../generate-third-party-notices.sh
```

Comment:
you can add a top level script for this that uses turbo for parallelization

Reply:
The way that I reworked the process makes it so that both actions are encapsulated within one bash script, so I won't be able to use Turbo for parallelization, unfortunately.  I was trying to figure out a way to do this all in the `yml` workflow and avoid bash scripts, but setting runtime variables (which is something that I needed to be able to do) was not as straightforward as I thought.

Reply:
I have a new approach where I don't think comment applies anymore, but LMK if you think it still does.  Mostly because I use a command that won't work correctly outside of GH actions since it requires the changing of workspaces to work correctly.